executive/Q5015503/current/popolo-m17n.json

```diff
@@ -34,103 +17,6 @@
 
       ]
     },
-    {
```

Comment:
These removed presidents are the result of me adding start and end dates to their historical P39s.